### PR TITLE
#27 - Cams behind webview based on z-index of CSS

### PR DESCRIPTION
**Pull request by mark-veenstra**
Added ability to load cams behind webview. With this PR old and new behaviour work all together.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,26 @@ All JavaScript code should be written in `onDeviceReady` function in `/js/index.
     }
 ```
 
+## Positioning the camera's behind or before webview
```
Comment (msach22) [inline README.md]: @mark-veenstra How do you feel about adding this information in the docs folder instead of the main repo readme?

Comment (mark-veenstra) [reply README.md]: That's probably better. So I moved it to the docs folder.

**Comment by msach22** [comment]
@mark-veenstra looks like there are some merge conflicts. Please resolve them so we can get this merged in!

**Comment by mark-veenstra** [comment]
@msach22 I pulled `3.2.0` over this branch and resolved the merge conflict. Result is pushed back to this branch

**Pull request merged**